.storybook/decorators/icons.js

```diff
@@ -36,66 +36,6 @@ import {
   faUserCircle as farUserCircle
 } from '@fortawesome/free-regular-svg-icons'
```

Comment:
@majid-amiri These icons were used in the stories for icons i think - it should not effect the bundling i think? please test those stories

Reply:
We decided to remove Pro icons from IUI.